src/smartpeak/source/ui/GraphicDataVizWidget.cpp

```diff
@@ -22,6 +22,7 @@
 // --------------------------------------------------------------------------
 
 #include <SmartPeak/ui/GraphicDataVizWidget.h>
+#include <SmartPeak/ui/FilePicker.h>
```

Comment:
I don't see you are using the FilePicker. it would be very nice to use it.

Reply:
It wasn't implemented by the time of reviewing, now it's in.